### PR TITLE
Update change log for v0.5.0 release

### PR DESCRIPTION
**Pull request by ccfishk**
Signed-off-by: jhu02 <jiminh@jiminhs-MacBook-Pro.local>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
 <a name="unreleased"></a>
 ## [Unreleased]
 
+<a name="v0.5.0"></a>
+## [v0.5.0] - 2022-03-08
+### Fixed
+- Build Fix (#102)
+### Changed
+- setup github actions as ci replacement.
```
Comment (kevdowney) [inline CHANGELOG.md:9]: Just remove this line, it's already captured below as `Migrate to GitHub Actions to gate PRs.`

**Comment by kevdowney** [comment]
The [version.go](https://github.com/keikoproj/addon-manager/blob/5c344ff49cdb94e3f82c073a427da15bf21ade4b/pkg/version/version.go#L22) file should also be updated to v0.5.0

File: CHANGELOG.md (hunk 1 above)
Comment (ccfishk) [reply CHANGELOG.md]: 0c23b57e6312e60bef51a98680a6f38b082eae44

**Comment by ccfishk** [comment]
> The [version.go](https://github.com/keikoproj/addon-manager/blob/5c344ff49cdb94e3f82c073a427da15bf21ade4b/pkg/version/version.go#L22) file should also be updated to v0.5.0
yes, v0.5.0
0c23b57e6312e60bef51a98680a6f38b082eae44

**Review by kevdowney — APPROVED** [review]

**Pull request merged**